addon/mixins/components/tooltips.js

```diff
@@ -62,7 +63,7 @@ export default Ember.Mixin.create({
   destroyTooltip: on('willDestroyElement', function() {
     const tooltip = this.get('tooltip');
 
-    if (tooltip && Ember.typeOf(tooltip) === 'object') {
+    if (tooltip && tooltip instanceof Tooltip) {
```

Comment:
Great, thanks for the update. I'll get on this ASAP so we can make another release - likely sometime this week.